src/wallet.d.ts

```diff
@@ -0,0 +1,58 @@
+declare module 'neon-js' {
+  type Account = {
+    address: string
+    privateKey: string
+    programHash: string
+    publicKeyEncoded: string
+    publicKeyHash: string
+  }
+
+  function addressToScriptHash(
```

Comment:
i think this is wrong. there is only 1 arg when looking at source code xD

Reply:
Good catch :)